tests/page-objects/login/login.po.js

```diff
@@ -47,5 +59,40 @@ module.exports = {
       await browser.wait(() => helper.isTextDisplayed(incorrectCredentialsText), 2000);
     }
   },
-  returnToLogin: () => element(by.css('.btn[href="/medic/login"]'))
+  returnToLogin: () => element(by.css('.btn[href="/medic/login"]')),
+  
+  getAllLocales: async () => {
+    const locales = await element.all(by.css('.locale')).map(loc => {
```

Comment:
Pull this into a function and reuse for `getAllLocales` and `getselectedLanguage`